databricks/koalas/series.py

```diff
@@ -4992,6 +4992,77 @@ def items(self) -> Iterable:
         """This is an alias of ``iteritems``."""
         return self.iteritems()
 
+    def tail(self, n=5):
```

Comment:
@itholic can we implement it in DataFrame here too? I thought the implementation will be pretty similar

Reply:
@HyukjinKwon Sounds good!